.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_IMAGE_URL="https://www.perseids.org/perseids-logo.png"
+REACT_APP_SITE_NAME="The Perseids Project"
+REACT_APP_TITLE="The Perseids Project"
+REACT_APP_DESCRIPTION="The Perseids Project aims to support access to scholarship in Classics for students and members of the public at all levels of competence. We provide a suite of tools that foster language acquisition, facilitate working with documents, and encourage research."
+REACT_APP_URL="https://www.perseids.org/"
```

Comment:
Just like in https://github.com/perseids-project/lsj-js/pull/15/files, we should customize the text a bit to fit the Greek Latin Demixer.

* `REACT_APP_IMAGE_URL`: should be `https://apps.perseids.org/demixer/perseids-logo.png`. This doesn't exist currently, but it will exist once we merge this PR and deploy it since you're adding that image in the `public/` folder.
* `REACT_APP_SITE_NAME`: we can keep this the same.
* `REACT_APP_TITLE`: let's change this to `Greek Latin Demixer`
* `REACT_APP_DESCRIPTION`: this should be a short description of the application. Maybe something like `The Greek and Latin Demixer is a tool that allows scholars to distinguish Greek and Latin characters in OCRed texts.` If you think of a better description or have a improvement on mine, feel free to use it.
* `REACT_APP_URL`: should be `https://apps.perseids.org/demixer/`